java/libs/src/main/java/org/physical_web/collection/PwsResult.java

```diff
@@ -302,49 +302,153 @@ public String getGroupId() {
 
   /**
    * Get extra boolean value.
+   * @param key The key of the stored value.
    * @return The stored value or null if it doesn't exist in specified form.
```

Comment:
The getExtra.* methods should say we'll throw JSONException if the key isn't present